e2e/testdata/violations/most_violations.rego

```diff
@@ -0,0 +1,110 @@
+# METADATA
+# description: |
+#   This file is used for e2e tests of most of the provided linter rules. All violations can't
+#   be tested in a single file, as some are mutually exlusive (i.e. implicit-future-keywords and
+#   rule-named-if).
+package all_violations
+
+# Imports
+
+# avoid-importing-input
+import input
+
+# implict-future-keywords
+import future.keywords
+
+# import-shadows-import
+import data.foo
+import data.foo
+
+import data.redundant.alias as alias
+
+# redundant-data-import
+import data
+
+### Bugs ###
+
+constant_condition {
+	1 == 1
+}
+
+# METADATA
+# invalid-metadata-attribute: true
+should := "fail"
+
+not_equals_in_loop {
+	"foo" != input.bar[_]
+}
+
+# rule-named-if
+# if { true }
```

Comment:
Should this be uncommented?

Reply:
https://github.com/StyraInc/regal/pull/192/files#diff-2fa353151f098139be7a7e83e4d4866fb7723b8d6833ce1b467cf69512be09f1R3-R5 isn't what it says here that it should be removed, as it needs its own file...? But I'm a bit confused myself 😅

Reply:
Yes it should be removed as it won’t trigger in this file due to the keywords import. Probably just forgot.

Reply:
👍 ahh I've removed it. I made another small fix in cce360e0cde331dadcb3d3f9ae13c0c6f27db472 after your comment.

Reply:
Ahh, ok! I'll update.

Reply:
af0966c022d9b3a383804e7b748387a58bc0b7f3 look ok?

Reply:
Ooops, https://github.com/StyraInc/regal/pull/194